cmd/ddev/cmd/config.go

```diff
@@ -198,108 +189,73 @@ func handleConfigRun(cmd *cobra.Command, args []string) {
 }
 
 func init() {
-	var err error
-
 	validAppTypes := strings.Join(ddevapp.GetValidAppTypes(), ", ")
 	projectTypeUsage := fmt.Sprintf("Provide the project type (one of %s). This is autodetected and this flag is necessary only to override the detection.", validAppTypes)
 	projectNameUsage := "Provide the project name of project to configure (normally the same as the last part of directory name)"
 
 	ConfigCommand.Flags().StringVar(&projectNameArg, "project-name", "", projectNameUsage)
 	ConfigCommand.Flags().StringVar(&docrootRelPathArg, "docroot", "", "Provide the relative docroot of the project, like 'docroot' or 'htdocs' or 'web', defaults to empty, the current directory")
-	ConfigCommand.Flags().StringVar(&composerRootRelPathArg, "composer-root", "", "Overrides the default Composer root directory for the web service")
-	ConfigCommand.Flags().BoolVar(&composerRootRelPathDefaultArg, "composer-root-default", false, "Unsets a web service Composer root directory override")
+	ConfigCommand.Flags().StringVar(&composerRootRelPathArg, "composer-root", "", "The relative path, from the project root, to the directory containing composer.json (This is where all Composer-related commands are executed.)")
+	ConfigCommand.Flags().BoolVar(&composerRootRelPathDefaultArg, "composer-root-default", false, `Unsets a web service Composer root directory override, the same as --composer-root=""`)
 	ConfigCommand.Flags().StringVar(&projectTypeArg, "project-type", "", projectTypeUsage)
```

Comment:
The only thing I didn't touch is the default value for `--project-type`. (We say in our docs that default is `php`.)